Assets/Prefabs/Enemies/brad_SpaceFly(Adult).prefab

```diff
@@ -0,0 +1,171 @@
+%YAML 1.1
```

Comment:
I would rename this to something simpler such as SpaceFlyAdult or AdultSpaceFly or something along those lines to make them easier to find